pxtblocks/blocklyloader.ts

```diff
@@ -2230,6 +2230,8 @@ namespace pxt.blocks {
             workspace.registerButtonCallback('CREATE_FUNCTION', function (button) {
                 let promptAndCheckWithAlert = (defaultName: string) => {
                     Blockly.prompt(newFunctionTitle, defaultName, function (newFunc) {
+                        // Tick event to monitor usage of "Make a function" button
+                        pxt.tickEvent('blocks.make_a_function');
```

Comment:
there isn't a naming convention for ticks, but we never use _, can you drop them?